RickLeinecker/Fortuna
# Write Casus Interpriter, Print blocks, List Append blocks

**Description**
This PR includes the entirety of the Casus Interpriter (mostly untested, although type safe and passes flow), the addition of several new blocks including Print and AppendToList. Although this will get thorough testing as soon as we have the battling working, I would be eternally grateful for solid code reviews on this (even though I know it is incredibly long). If you see things that look like they are bugs or oversights I made, don't hesitate to let me know :)

This also causes Issue #81. I'll fix this once we have some way of running the Casus code. For now, the priority is correctness, and then we can optimize to ~O(1) once we know everything works properly.

**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
Haha yeah that's the thing... You can't really do this until we have some way to run this from the UI.
You can run flow to verify that there aren't any majorly broken parts.

**Final Checklist:**
Go down the list and check them off.

- [ x I have performed a self-review of my own code
- [na] I have made corresponding changes to the design document (if applicable)
